webrtc/videoroom/assets/src/room_ui.ts

```diff
@@ -45,27 +45,35 @@ export function getRoomId(): string {
   return document.getElementById("room")!.dataset.roomId!;
 }
 
-function elementId(stream: MediaStream, type: "video" | "audio" | "feed") {
-  return `${type}-${stream.id}`;
+function elementId(streamId: string, type: "video" | "audio" | "feed") {
+  return `${type}-${streamId}`;
 }
 
 export function addVideoElement(
   stream: MediaStream,
   label: string,
-  mute: boolean = false
+  muted: boolean = false
 ): void {
-  const id = elementId(stream, "video");
-  let video = document.getElementById(id) as HTMLVideoElement;
-
-  if (!video) {
-    video = setupVideoFeed(stream, label);
+  const videoId = elementId(stream.id, "video");
+  const audioId = elementId(stream.id, "audio");
+  let video = document.getElementById(videoId) as HTMLVideoElement;
+  let audio = document.getElementById(audioId) as HTMLAudioElement;
+  if (!video && !audio) {
+    const values = setupVideoFeed(stream, label);
+    video = values.video;
+    audio = values.audio;
   }
 
-  video.id = id;
+  video.id = videoId;
   video.srcObject = stream;
   video.autoplay = true;
   video.playsInline = true;
-  video.muted = mute;
+  video.muted = true;
+
+  audio.id = audioId;
```

Comment:
Aren't we playing audio twice here? Once in video element and the other time in audio element? We probably don't need audio element anymore

Reply:
Audio element is needed to hear a person that has not been displayed yet. In other case we have to wait for its video tile. This way we lose about 1 sec of speech. We also always mute video element to not play audio twice

Reply:
Btw. version without audio element is available in this commit 7a4dfc126592b9a21d23454e48bb43b0c696b0b7 if you wanted to test it